### ISSUE TITLE
Encryption key integrity check failed.

### ISSUE THREAD

**Issue by computeronix** [Storj/libstorj#317]
When I logged into the bridge through the storj.exe app (version: libstorj-1.0.0-rc3)

I had it create a new encryption key for me and below are the values i specified and at the very end it said Encryption Key Integrity check failed.

```
If you've previously uploaded files, please enter your existing encryption key (12 to 24 words).
Otherwise leave the field blank to generate a new key.

Encryption key:

We now need to create an secret key used for encrypting files.
Please choose strength from: 128, 160, 192, 224, 256

Strength: 224

Encryption key: cloud step space place regret exile math corn subject labor that real damage snake hire laugh ring arrange hover copy rib

Please make sure to backup this key in a safe location. If the key is lost, the data uploaded will also be lost.

Encryption key integrity check failed.
```

**Comment by braydonf** [comment]
Looks like there is an issue with 160 and 224

**Comment by braydonf** [comment]
Included in `v1.0.0-rc4`
